app/components/projectAssignment/workWeekInput.tsx

```diff
@@ -11,6 +11,7 @@ interface WorkWeekInputProps {
 	assignment?: AssignmentType;
 	cweek: number | null;
 	year: number | null;
+	isUserTBD:boolean;
```

Comment:
nit: space between colon and type? We should probably set up some linting rules for things like this.

Reply:
added space